zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/socket/Message.java

```diff
@@ -127,8 +127,8 @@
     APP_STATUS_CHANGE,      // [s-c] on app status change
 
     LIST_NOTEBOOK_JOBS,     // [c-s] get notebook job management infomations
-    LIST_UPDATE_NOTEBOOK_JOBS, // [c-s] get job management informations for until unixtime
-                               // @param unixTime
+    LIST_UPDATE_NOTEBOOK_JOBS, // [c-s] get job management informations
```

Comment:
`LIST_UPDATE_NOTEBOOK_JOBS` seems server -> client message. Could you update comment `[s-c]` ?